test/Handler/HomeSpec.hs

```diff
@@ -9,7 +9,7 @@ spec = withApp $ do
         it "loads the index and checks it looks right" $ do
           get HomeR
           statusIs 200
-          htmlAllContain "h1" "Welcome to Yesod"
+          htmlAnyContain "h1" "Yesod; A modern framework for blazing fast websites"
```

Comment:
Correct, I have fixed it. The tests still fail, but on what seems to look like an unrelated issue.